_format/1.1/index.md

```diff
@@ -585,6 +596,96 @@ that includes a [profile alias][profile aliases]:
 > Note: Additional link types, similar to `profile` links, may be specified in
 the future.
 
+#### <a href="#document-links-link-relations" id="document-links-link-relations" class="headerlink"></a> Link relation types
+
+The following link relation types **MAY** be used as the value of the `rel` link
+parameter object member:
+
+  * `add`: The link's target points to a resource where a user agent can cause
+    an addition to the link's context.
+  * `update`: The link's target points to a resource where a user agent can cause
+    an update the link's context.
+  * `remove`: The link's target points to a resource where a user agent can cause
```

Comment:
https://jsonapi.org/format/#crud says "Creating, updating and deleting". Here, we say `add`, `update` and `remove`. The first and the last deviate from the terminology used by the spec today. Am I right in assuming this is to avoid these words from matching HTTP method names?